aws-s3-transfer-manager/src/operation/download_objects.rs

```diff
@@ -43,6 +43,16 @@ impl DownloadObjects {
         let destination = input.destination().expect("destination set");
         validate_destination(destination).await?;
 
+        // create span to serve as parent of spawned child tasks
+        let parent_span_for_tasks = tracing::debug_span!(
+            parent: None,
```

Comment:
Don't we want the parent to be whatever the current span is? Whatever span (if any) is in the current context  when calling `orchestrate` is the parent span. 

If the parent child relationship is correct we shouldn't need to add the `follows_from` for this span (also `[follows_from](https://docs.rs/tracing/latest/tracing/struct.Span.html#method.follows_from)` implies a indirect causal relationship, is that correct? Seems like a normal parent/child relationship would be appropriate. See [docs for example scenario](https://github.com/tokio-rs/tracing/blob/tracing-0.1.40/tracing/src/span.rs#L177-L193)). I could see possibly the upload worker tasks using this though depending.

Reply:
A span doesn't "end" until all its children end. So if the spawned-tasks are children of the initial `tm.download().send()` function, it would make that `send()` span look waaaaay longer than the length of time `send().await` actually takes.

If API was just a single async `send()` function, I would make the spawned tasks be its children. But since our API requires the user to call `send()`, then `join()` / `collect()`, it's not clear which span the spawned-tasks should be children of.